test/integration/first-operator-test/01-scale.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
```

Comment:
Is the correct way to scale to update the deployment replicas directly?

Reply:
I'm following https://kudo.dev/docs/testing.html#writing-your-first-test here.

Reply:
The example shows how to adjust the replicas on a deployment when testing a deployment. You aren’t testing a deployment so would be writing a different test than the example...